lib/Imagine/Gd/Drawer.php

```diff
@@ -292,8 +421,13 @@ private function getColor(ColorInterface $color)
             throw new InvalidArgumentException('GD driver only supports RGB colors');
         }
 
-        $gdColor = imagecolorallocatealpha($this->resource, $color->getRed(), $color->getGreen(), $color->getBlue(), (100 - $color->getAlpha()) * 127 / 100);
-        if (false === $gdColor) {
+        if (false === ($gdColor = imagecolorallocatealpha(
```

Comment:
For better readability, I'd prefer to keep the assignment and the `if` in two separate lines.